app/lib/controllers/todo_controller.dart

```diff
@@ -306,6 +308,26 @@ class ToDoController extends GetxController {
     );
   }
 
+  // Mock data for subscribed users
+  List<SubscriberModel> listSubscribers = <SubscriberModel>[
+    SubscriberModel('', 'Okon Invincible', false),
+    SubscriberModel('', 'Floym Dore', false),
+    SubscriberModel('', 'Raveena Tondon', false),
+    SubscriberModel('', 'Karishma Kapoor', false),
+    SubscriberModel('', 'Hema Malini', false),
+    SubscriberModel('', 'John Doe', false),
+  ];
+
+  void handleCheckClick(int position) {
```

Comment:
There is already defined function which is handling check/uncheck functionality.

```
  bool toggleCheck(int idx, ToDoTaskItem item) {
    if (item.isCompleted == true) {
      ToDoTaskItem newItem = ToDoTaskItem(
        title: item.title,
        isCompleted: false,
        hasMessage: item.hasMessage,
        dateTime: item.dateTime,
        subtitle: item.subtitle,
        notes: item.notes,
        lastUpdated: item.lastUpdated,
        toggleCompletion: (w) => toggleCheck(idx, w),
      );
      tasks[idx].remove(item);
      tasks[idx].add(newItem);
      return false;
    } else {
      ToDoTaskItem newItem = ToDoTaskItem(
        title: item.title,
        isCompleted: true,
        hasMessage: item.hasMessage,
        dateTime: item.dateTime,
        subtitle: item.subtitle,
        notes: item.notes,
        lastUpdated: item.lastUpdated,
        toggleCompletion: (w) => toggleCheck(idx, w),
      );
      tasks[idx].remove(item);
      tasks[idx].add(newItem);
      return true;
    }
  }
  ```